sdk/servicebus/azure-servicebus/samples/async_samples/batch_delete_async.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show sending message(s) to a Service Bus Queue.
+"""
+
+import os
+import asyncio
+from datetime import datetime, timezone, timedelta
+from azure.servicebus.aio import ServiceBusClient
+from azure.servicebus import ServiceBusMessage
+
+
+CONNECTION_STR = os.environ['SERVICEBUS_CONNECTION_STR']
+QUEUE_NAME = os.environ["SERVICEBUS_QUEUE_NAME"]
+
+async def send_single_message(sender, i):
+    message = ServiceBusMessage(f"Single Message {i}")
+    await sender.send_messages(message)
+
+async def run():
+    servicebus_client = ServiceBusClient.from_connection_string(conn_str=CONNECTION_STR)
+    async with servicebus_client:
+        sender = servicebus_client.get_queue_sender(queue_name=QUEUE_NAME)
+        async with sender:
+            await send_single_message(sender, 1)
+            await send_single_message(sender, 2)
+            await send_single_message(sender, 3)
+        print(f"All messages sent before {datetime.now(timezone.utc)}")
+
+
+        receiver = servicebus_client.get_queue_receiver(queue_name=QUEUE_NAME)
+        async with receiver:
+
+            # Peek before deleting to see enqueued times
+            peeked_msgs = await receiver.peek_messages(max_message_count=10)
+            for msg in peeked_msgs:
+                print(f"Message peeked has enqueued time of {msg.enqueued_time_utc}")
```

Comment:
since we peek to check this, would we either want to use this peeked enqueued time in any of the deletes? o/w maybe remove this part since it doesn't seem directly connected to the delete op?